frontend/src/components/Secrets.js

```diff
@@ -0,0 +1,48 @@
+import React, {useState} from 'react' ;
+
+export const Secrets = () => {
+  const [authStatus, setAuthStatus] = useState('')
+  const [secret, setSecret] = useState('')
+
+  const accessToken = window.localStorage.getItem('accessToken')
+  console.log("Access token from localStorage: " + accessToken)
+
+  // Get secret from back-end. Here we use GET and not POST (see back end code)
+  fetch('https://projectauthannaochelin.herokuapp.com/secrets', {

```

Comment:
It's really useful to make these URLs `const` with a good name so you can easily change them and avoid misspelling